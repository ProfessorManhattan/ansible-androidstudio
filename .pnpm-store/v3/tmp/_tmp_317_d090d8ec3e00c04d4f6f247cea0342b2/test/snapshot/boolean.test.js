const parser = require("../main");

describe("boolean", () => {
  it("assign", () => {
    expect(parser.parseEval("$var = true;")).toMatchSnapshot();
  });
  it("assign (2)", () => {
    expect(parser.parseEval("$var = false;")).toMatchSnapshot();
  });
});
