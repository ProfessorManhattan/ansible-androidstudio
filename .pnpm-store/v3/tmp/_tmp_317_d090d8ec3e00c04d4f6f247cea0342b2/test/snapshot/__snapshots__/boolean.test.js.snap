// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`boolean assign (2) 1`] = `
Program {
  "children": Array [
    ExpressionStatement {
      "expression": Assign {
        "kind": "assign",
        "left": Variable {
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
        "operator": "=",
        "right": Boolean {
          "kind": "boolean",
          "raw": "false",
          "value": false,
        },
      },
      "kind": "expressionstatement",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`boolean assign 1`] = `
Program {
  "children": Array [
    ExpressionStatement {
      "expression": Assign {
        "kind": "assign",
        "left": Variable {
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
        "operator": "=",
        "right": Boolean {
          "kind": "boolean",
          "raw": "true",
          "value": true,
        },
      },
      "kind": "expressionstatement",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
